src/styles/main.scss

```diff
@@ -198,3 +198,10 @@ li em{
     padding: 5px 15px;
   }
 }
+
+// For nav elements that have border-radius and left/right border
+.removeNavBorderStyles {
```

Comment:
we're not following camelCase convention for css, `remove-nav-border-styles` or `no-nav-border` would be much better name.